### ISSUE TITLE
console config doesn't allow for converted pre-Composer site

### ISSUE THREAD

**Issue by narration-sd** [craftcms/cms#1579]
### Description
This is pretty straigntforward - the CRAFT_BASE_PATH will be wrong in craft.php, needing /craft added to it. 

Something that could be set up in the conversion script?

### Steps to reproduce

1. Have a site converted from pre-Composer days, so that folders are of the old layout.
2. try to run the craft.php console, and find you can't.

### Additional info

PHP version	7.1.2-4+deb.sury.org~xenial+1
Database driver & version	MySQL 5.7.17-0ubuntu0.16.04.1
Image driver & version	GD 7.1.2-4+deb.sury.org~xenial+1
Craft edition & version	Craft Personal 3.0.0-beta.9
Yii version	2.0.11.2
Twig version	2.1.0
Guzzle version	6.2.1
Imagine version	0.7-dev
Plugins

Trimmer	1.0.0
Bactrian	1.0.0
Standards Agent	2.0.4

**Comment by brandonkelly** [comment]
Which `craft.php` are you referring to? Craft hasn’t ever shipped with one.

**Comment by narration-sd** [comment]
- uh, I think you're right about no craft.php -- I might have renamed 'craft' to craft.php at  some point early in this conversion, not sure why as it has a hash-bang to call php anyway.

- so my point would be in this 'craft' executale at the top folder for a converted site, not a fresh-from-composer new site, will not have the right CRAFT_BASE_PATH value

- but, I have by now just about forgotten over the many months exactly what history this site went through. I believe it was originally up-converted from a Craft 2 site --in any case, it's a dead copy of one, my favorite try-many-things basis. There is some vague thought this didn't work/had problems at some early Craft 3 time. This might just be about those missing typeId/tightened-up foreign keys consistency issues, which I sorted out at the time by manually assigning them, in conversation I think wiith Brad.

Summary of this issue: if Craft 3 yet/intentionally/soon does up-conversions from Craft 2, leaving the original site folder structure intact as mine is,  you'd need to use a differing CRAFT_BASE_PATH in the 'craft' console runner, to fit. I just changed mine, of course. To:

`define('CRAFT_BASE_PATH', $root . "/craft");
`

Just checked between fresh and converted sites to verify.
